staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -625,6 +626,7 @@ type fancyResponseWriterDelegator struct {
 }
 
 func (f *fancyResponseWriterDelegator) CloseNotify() <-chan bool {
```

Comment:
Mark this fancyResponseWriterDelegator.CloseNotify() as deprecated?

Reply:
Done.